qiskit/pulse/instructions/frequency.py

```diff
@@ -12,33 +12,32 @@
 # copyright notice, and modified files need to carry a notice indicating
 # that they have been altered from the originals.
 
-"""
-Frequency instructions module. These instructions allow the user to manipulate
+"""Frequency instructions module. These instructions allow the user to manipulate
 the frequency of a channel.
 """
-from typing import List, Optional, Union
+from typing import Optional, Tuple
 
 from ..channels import PulseChannel
 from .instruction import Instruction
 
 
 class SetFrequency(Instruction):
-    """Set the channel frequency. This command operates on PulseChannels.
+    r"""Set the channel frequency. This command operates on PulseChannels.
     A PulseChannel creates pulses of the form
 
     .. math::
-        Re[exp(i 2pi f jdt + phase) d_j].
+        Re[\exp(i 2\pi f jdt + \phi) d_j].
 
-    Here, f is the frequency of the channel. The command SetFrequency allows
-    the user to set the value of f. All pulses that are played on a channel
+    Here, *f* is the frequency of the channel. The command SetFrequency allows
+    the user to set the value of *f*. All pulses that are played on a channel
     after SetFrequency has been called will have the corresponding frequency.
 
     The duration of SetFrequency is 0.
     """
 
     def __init__(self, frequency: float,
                  channel: Optional[PulseChannel],
-                 name: Optional[str] = None):
+                 name: Optional[str] = 'setfrequency'):
```

Comment:
yes it's just a default display name

Reply:
since it shows up in a plot, it could be useful to see what kind of instruction it is, but if the user isn't supplying the name themselves, anything we add won't be helpful for them to pick out different insts. I've also added `id` which gives each Instruction instance a unique id (per program)

Reply:
I would argue that if this is not a unique identifier name, and is therefore only for display purposes. If not set by the user it shouldn't be displayed at all and this should, therefore, be ``None``

Reply:
ok! sounds great!